Scarb.toml

```diff
@@ -8,7 +8,7 @@ cairo-version = ">=2.0.0-rc0"
 sierra-replace-ids = true
 
 [dependencies]
-dojo = { git = "https://github.com/dojoengine/dojo", tag = "nightly-e7abee5"}
+dojo = { git = "https://github.com/dojoengine/dojo", tag = "nightly"}
```

Comment:
why this specifically and not the latest one [`nightly-9030d68dc1c4bef4a666ded520a953f145d9aa2a`](https://github.com/dojoengine/dojo/releases/tag/nightly-9030d68dc1c4bef4a666ded520a953f145d9aa2a)?